test/t8n/t8n.cpp

```diff
@@ -255,7 +255,11 @@ int main(int argc, const char* argv[])
             // EIP-7685: General purpose execution layer requests
             j_result["requests"] = json::json::array();
             for (size_t i = 0; i < requests.size(); ++i)
-                j_result["requests"][i] = hex0x(requests[i].data());
+            {
+                if (!requests[i].data().empty())
+                    // Skip empty requests, include the type byte
```

Comment:
Where is the type byte added?

Reply:
`raw_data` includes it (next line). Do we want to concatenate explicitly here?

Reply:
Ok, so these are two comments?
1. Skip empty requests
2. Include non-empty requests with the  leading type byte.

Reply:
yes... maybe this is over-commented, but I just mirrored the other similar `if` in the hashing routine

Reply:
Just split the comment: one part next to the `if`, other part before the `emplace_back`.

Reply:
Or "Only report non-empty requests. Include the leading type byte."